src/clang-c-frontend/clang_c_main.cpp

```diff
@@ -70,18 +70,37 @@ bool clang_c_maint::clang_main()
     // if user provided class/contract name
     if (!config.cname.empty() && !config.main.empty())
     {
-      const std::string fmt = "@" + config.cname + "@F@" + main;
-      if (it->second.as_string().find(fmt) == std::string::npos)
-        continue;
+      // Allow user to provide a range. e.g.
+      // --class "A B" --function test
+      // --contract "Base Derieve" --function _ESBMC_main
+      // yet only the first matched one will get verified
```

Comment:
Why will only the first match be verified? What is the purpose of this range?

Reply:
Also, since esbmc now supports multi-property, we might extend the `--function` to allow multiple inputs. Like `--function ''test1 test2"`